source/extensions/http/header_validators/envoy_default/http1_header_validator.cc

```diff
@@ -166,16 +172,6 @@ ValidationResult Http1HeaderValidator::validateRequestHeaders(const RequestHeade
   }
 
   // Step 2: Validate Transfer-Encoding and Content-Length headers.
-  // HTTP/1.1 disallows a Transfer-Encoding and Content-Length headers,
```

Comment:
nit: Let's move this comment into validateContentLengthAndTransferEncoding in order to make clear why that method returns reject.

Reply:
done